drivers/sensor/lis2dw12/lis2dw12_trigger.c

```diff
@@ -67,6 +67,19 @@ static int lis2dw12_enable_int(const struct device *dev,
 		return lis2dw12_pin_int1_route_set(ctx,
 				&int_route.ctrl4_int1_pad_ctrl);
 #endif /* CONFIG_LIS2DW12_TAP */
+#ifdef CONFIG_LIS2DW12_THRESHOLD
+	/**
+	 * Trigger fires when channel reading transitions configured
+	 * thresholds.  The thresholds are configured via the @ref
+	 * SENSOR_ATTR_LOWER_THRESH and @ref SENSOR_ATTR_UPPER_THRESH
+	 * attributes.
+	 */
+	case SENSOR_TRIG_THRESHOLD:
+		LOG_INF("Setting int1_wu: %d\n", enable);
```

Comment:
You get the idea about LOG_INF being probably too noisy -- I'll stop mentioning it